pom.xml

```diff
@@ -1443,7 +1446,7 @@
 
     <!-- NOTE: When updating GeoTools, check which version
     of Postgres is used and update pg.version if needed. -->
-    <geotools.version>22.3</geotools.version>
+    <geotools.version>23.0</geotools.version>
```

Comment:
We should build once more when GeoTools is officially anounced, we are currently having a build issue due to the new osgeo repo ... performing faster and allowing multi-threaded builds to trip on themselves. The Maven local repository is not thread safe apparently ...